config/config.json

```diff
@@ -9,6 +9,25 @@
     "noise diode powered" : "relay2=1",
     "antenna path enabled": "relay1=0",
     "noise diode path enabled": "relay1=1"
+  },
+  "sensors": {
+    "internal_temp": 1,
+    "internal_humidity": 2,
+    "tec_intake_temp": 3,
+    "tec_exhaust_temp": 4
+  },
+  "digital_inputs": {
+      "ups_power": 1,
+      "ups_battery_level": 2,
+      "ups_trouble": 3,
+      "ups_battery_replace": 4
+  },
+  "analog_inputs": {
+      "door_sensor": 1,
```

Comment:
This is OK here as an example, but just in case this makes its way into actual configs: `"door_sensor"` should be a digital input.